src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -1,111 +1,84 @@
 package io.iohk.ethereum.blockchain.sync
 
-import scala.concurrent.duration._
-import scala.concurrent.ExecutionContext.Implicits.global
-import akka.actor.SupervisorStrategy.Stop
-import akka.actor._
-import akka.util.ByteString
-import io.iohk.ethereum.db.storage._
-import io.iohk.ethereum.domain._
+import akka.actor.{Actor, ActorLogging, ActorRef, PoisonPill, Props, Scheduler}
+import io.iohk.ethereum.db.storage.{AppStateStorage, FastSyncStateStorage}
+import io.iohk.ethereum.domain.Blockchain
 import io.iohk.ethereum.ledger.Ledger
-import io.iohk.ethereum.network.EtcPeerManagerActor.PeerInfo
-import io.iohk.ethereum.network.PeerEventBusActor.PeerEvent.PeerDisconnected
-import io.iohk.ethereum.network.PeerEventBusActor.SubscriptionClassifier.PeerDisconnectedClassifier
-import io.iohk.ethereum.network.PeerEventBusActor.{PeerSelector, Subscribe, Unsubscribe}
-import io.iohk.ethereum.network.p2p.messages.PV62.BlockBody
-import io.iohk.ethereum.network.{EtcPeerManagerActor, Peer, PeerId}
 import io.iohk.ethereum.utils.Config.SyncConfig
 import io.iohk.ethereum.validators.Validators
 
 class SyncController(
-    val appStateStorage: AppStateStorage,
-    val blockchain: Blockchain,
-    val fastSyncStateStorage: FastSyncStateStorage,
-    val ledger: Ledger,
-    val validators: Validators,
-    val syncConfig: SyncConfig,
-    val peerEventBus: ActorRef,
-    val pendingTransactionsManager: ActorRef,
-    val ommersPool: ActorRef,
-    val etcPeerManager: ActorRef,
-    val externalSchedulerOpt: Option[Scheduler] = None)
+    appStateStorage: AppStateStorage,
+    blockchain: Blockchain,
+    fastSyncStateStorage: FastSyncStateStorage,
+    ledger: Ledger,
+    validators: Validators,
+    peerEventBus: ActorRef,
+    pendingTransactionsManager: ActorRef,
+    ommersPool: ActorRef,
+    etcPeerManager: ActorRef,
+    syncConfig: SyncConfig,
+    externalSchedulerOpt: Option[Scheduler] = None)
   extends Actor
-    with ActorLogging
-    with BlacklistSupport
```

Comment:
Yes, it's true, the lists are now separate. I thought, since they're quite volatile anyway (200s for blacklist now), it wouldn't really matter.